ehitchhiking-ui/src/app/shared/components/trip/trip.component.html

```diff
@@ -0,0 +1,38 @@
+<div class="trips" >
+<div
+  class="trips-user"
+>
+  <i class="material-icons">
+    directions_car
+  </i>
+  <p class="trips-list truncated">{{trip.startPoint}}  ->  {{trip.endPoint}}</p>
+  <div class = "form-row">
+    <button class = "trip-btn" >
+      <i class="material-icons" *ngIf="trip.status === 0" title="active trip">
```

Comment:
Pls use ng-switch instead of three ng-if

Reply:
okay